### PR TITLE
fix error when there's no CustomField in the original testrun

### PR DESCRIPTION
**Pull request by pruan-rht**
fixes JIRA https://issues.redhat.com/browse/OCPQE-3399

### CODE REVIEW

**Comment by pruan-rht** [comment]
@mffiedler fixed now ```$ ./polarshift.rb clone-run 20210218-0023
waiting for operation up to 360 seconds..
[23:58:54] INFO> HTTP GET user@https://polarshift-api-aosqe.apps.ocp4.prod.psi.redhat.com/project/OSE/run/20210218-0023
[23:58:58] INFO> HTTP GET took 4.496 sec: 200 OK | application/json 737025 bytes

[23:58:58] INFO> HTTP POST user@https://polarshift-api-aosqe.apps.ocp4.prod.psi.redhat.com/project/OSE/run
[23:58:59] INFO> HTTP POST took 0.247 sec: 202 Accepted | application/json 267 bytes

[23:58:59] INFO> to check operation status manually, you can: curl 'https://polarshift-api-aosqe.apps.ocp4.prod.psi.redhat.com/polarion/request/6074dec3387b030011097a21' -u user:thepassword
waiting for operation up to 360 seconds..
[23:58:59] INFO> HTTP GET user@https://polarshift-api-aosqe.apps.ocp4.prod.psi.redhat.com/polarion/request/6074dec3387b030011097a21
[23:58:59] INFO> HTTP GET took 0.237 sec: 200 OK | application/json 122 bytes

[23:59:14] INFO> HTTP GET user@https://polarshift-api-aosqe.apps.ocp4.prod.psi.redhat.com/polarion/request/6074dec3387b030011097a21
[23:59:14] INFO> HTTP GET took 0.238 sec: 200 OK | application/json 595 bytes

Source run id '20210218-0023' cloned to new run id: '20210412-2358'```

**Comment by pruan-rht** [comment]
@liangxia @mffiedler PTAL and merge if there's are  issues

File: tools/polarshift.rb [hunk 1]
```diff
@@ -469,9 +469,13 @@ def transform_custom_fields(src)
     def clone_run(test_run_id: nil)
       template_hash = {}
       query_result = polarshift.get_run_smart(project, test_run_id)
-
       tc_ids = extract_test_case_ids(query_result)
-      custom_fields = transform_custom_fields(query_result['customFields']['Custom'])
+      if query_result['customFields'].nil?
+        # hard-code it to something
+        custom_fields = { "caseimportance" => "critical", "description" => "" }
```
Comment (liangxia) [inline tools/polarshift.rb:475]: Hard code case importance to critical is expected ?

Comment (pruan-rht) [reply tools/polarshift.rb:475]: @liangxia it just need something there.  I just took that from a working example.  Don't know what are the available options

Comment (akostadinov) [reply tools/polarshift.rb:475]: @pruan-rht , how about defaulting to an empty hash?

**Pull request merged**